### PR TITLE
continue updating translation

### PR DESCRIPTION
**Pull request by AlwaysReVaule**
After PR: https://github.com/mdn/translated-content/pull/5769#issue-1244138657
Relate PR: https://github.com/mdn/translated-content/pull/5756#issue-1243876007

### CODE REVIEW

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html [hunk 1]
```diff
@@ -243,15 +241,14 @@ <h2 id="ECMAScript_2015_Classes">ECMAScript 2015 Classes</h2>
...
-	<li><code>greeting()</code> and <code>farewell()</code> are class methods. Any methods you want associated with the class are defined inside it, after the constructor. In this example, we've used <a href="/en-US/docs/Web/JavaScript/Reference/Template_literals">template literals</a> rather than string concatenation to make the code easier to read.</li>
+	<li><code><a href="/en-US/docs/Web/JavaScript/Reference/Classes/constructor">constructor()</a></code>方法定义了一个构造函数来展示我们的<code>Person</code>类。</li>
+	<li><code>greeting()</code>和<code>farewell()</code> 是类方法。任何你想要和这个类联系的方法都被定义在其中，在构造器之后。在这个例子里，我们使用了<a href="/en-US/docs/Web/JavaScript/Reference/Template_literals">template literals</a>而不是字符串连接去使代码更易读。</li>
 </ul>
 
-<p>We can now instantiate object instances using the <a href="/en-US/docs/Web/JavaScript/Reference/Operators/new"><code>new</code> operator</a>, in just the same way as we did before:</p>
+<p>我们现在可以使用<a href="/en-US/docs/Web/JavaScript/Reference/Operators/new"><code>new</code>运算符</a>去实例化对象实例，就像我们之前做过的一样的方式:</p>
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:251]:
```suggestion
<p>我们现在可以使用 <a href="/zh-CN/docs/Web/JavaScript/Reference/Operators/new"><code>new</code> 运算符</a>来实例化对象实例，就像我们之前使用过的方式一样：</p>
```

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html [hunk 2]
```diff
@@ -263,7 +260,7 @@ <h2 id="ECMAScript_2015_Classes">ECMAScript 2015 Classes</h2>
 </pre>
 
 <div class="note">
-<p><strong>Note</strong>: Under the hood, your classes are being converted into Prototypal Inheritance models — this is just syntactic sugar. But I'm sure you'll agree that it's easier to write.</p>
+<p><strong>注意</strong>：在引擎盖下，您的类正在转换为原型继承模型 - 这只是语法糖。但我相信你会同意它更容易写。</p>
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:263]: 这句话是否过于“直译”了：

```suggestion
<p><strong>注意</strong>：在 JS 引擎下，类会被转换为原型继承模型——这只是语法糖。但我相信你会觉得这是一种更容易的写法。</p>
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html (hunk 2 above)
Comment (AlwaysReVaule) [reply files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:263]: @yin1999 是的，当时我很困惑这个hood要怎么译

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html (hunk 1 above)
Comment (AlwaysReVaule) [reply files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:251]: 下次我会注意引用连接自中文简体 😊 @yin1999

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html [hunk 3]
```diff
@@ -208,21 +208,19 @@ <h2 id="Object_member_summary">对象成员总结</h2>
  <li>那些定义在构造器函数中，用于给予对象实例的属性或方法。它们都很容易发现——在您自己的代码中，它们是构造函数中使用<code>this.x = x</code>类型的行；在浏览器内已经构建好的代码中，它们是可用于对象实例的成员（这些对象实例通常使用<code>new</code>关键字调用构造函数来创建，例如<code>var myInstance = new myConstructor()</code>）。</li>
  <li>那些直接在构造函数上定义，仅在构造函数上可用的属性或方法。它们通常仅在浏览器的内置对象中可用，并通过被直接链接到构造函数来识别，而不是实例。例如<code><a href="/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/keys">Object.keys()</a></code>。<strong>它们一般被称作静态属性或静态方法</strong>。</li>
  <li>那些在构造函数原型上定义，由所有实例和对象类继承的属性或方法。它们包括在构造函数的原型属性上定义的任何成员，如<code>myConstructor.prototype.x()</code>。</li>
- <li>Those available on an object instance, which can either be an object created when a constructor is instantiated like we saw above (so for example <code>let teacher1 = new Teacher( 'Chris' );</code> and then <code>teacher1.name</code>), or an object literal (<code>let teacher1 = { name : 'Chris' }</code> and then <code>teacher1.name</code>).</li>
+ <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如<code>let teacher1 = new Teacher( 'Chris' );</code>  然后和 <code>teacher1.name</code>)，也可以是一个对象文字（<code>let teacher1 = { name : 'Chris' }</code>然后和<code>teacher1.name</code>）。</li>
```
[the remark this answers is not in the document]
Comment (AlwaysReVaule) [reply files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:211]: 😊好的 @yin1999

**Comment by irvin** [comment]
please resolve conflict

**Comment by AlwaysReVaule** [comment]
> please resolve conflict

@irvin The conflict already resolved. 😊

**Comment by AlwaysReVaule** [comment]
> please resolve conflict

@irvin The conflict already resolved. 😊

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html [hunk 4]
```diff
@@ -208,17 +208,17 @@ <h2 id="Object_member_summary">对象成员总结</h2>
  <li>那些定义在构造器函数中，用于给予对象实例的属性或方法。它们都很容易发现——在您自己的代码中，它们是构造函数中使用<code>this.x = x</code>类型的行；在浏览器内已经构建好的代码中，它们是可用于对象实例的成员（这些对象实例通常使用<code>new</code>关键字调用构造函数来创建，例如<code>var myInstance = new myConstructor()</code>）。</li>
  <li>那些直接在构造函数上定义，仅在构造函数上可用的属性或方法。它们通常仅在浏览器的内置对象中可用，并通过被直接链接到构造函数来识别，而不是实例。例如<code><a href="/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/keys">Object.keys()</a></code>。<strong>它们一般被称作静态属性或静态方法</strong>。</li>
  <li>那些在构造函数原型上定义，由所有实例和对象类继承的属性或方法。它们包括在构造函数的原型属性上定义的任何成员，如<code>myConstructor.prototype.x()</code>。</li>
- <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如<code>let teacher1 = new Teacher( 'Chris' );</code>，然后是 <code>teacher1.name</code>），也可以是一个对象文本（<code>let teacher1 = { name : 'Chris' }</code>，然后是<code>teacher1.name</code>）。</li>
+ <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如 <code>let teacher1 = new Teacher( 'Chris' );</code>  以及其属性 <code>teacher1.name</code>)，也可以是一个对象字面量（<code>let teacher1 = { name : 'Chris' }</code> 以及其属性 <code>teacher1.name</code>）。</li>
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:211]: 多了个空格：

```suggestion
 <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如 <code>let teacher1 = new Teacher( 'Chris' );</code> 以及其属性 <code>teacher1.name</code>)，也可以是一个对象字面量（<code>let teacher1 = { name : 'Chris' }</code> 以及其属性 <code>teacher1.name</code>）。</li>
```

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html [hunk 5]
```diff
@@ -208,17 +208,17 @@ <h2 id="Object_member_summary">对象成员总结</h2>
...
 </ol>
 
 <p>如果您现在觉得一团浆糊，别担心——您现在还处于学习阶段，不断练习才会慢慢熟悉这些知识。</p>
 
-<h2 id="ECMAScript_2015_Classes">ECMAScript 2015 Classes</h2>
-
-<p>ECMAScript 2015 introduces <a href="/en-US/docs/Web/JavaScript/Reference/Classes">class syntax</a> to JavaScript as a way to write reusable classes using easier, cleaner syntax, which is more similar to classes in C++ or Java. In this section we'll convert the Person and Teacher examples from prototypal inheritance to classes, to show you how it's done.</p>
+<h2 id="ECMAScript_2015_Classes">ECMAScript 2015 类</h2>
 
+<p>ECMAScript 2015 引入了<a href="/en-US/docs/Web/JavaScript/Reference/Classes">class syntax</a>到 JavaScript，作为使用更简单、更简洁的语法编写可重用类的一种方式，这更类似于 C++ 或 Java 中的类。在本节中，我们将个人和教师示例从原型继承转换为类，以向你展示它是如何完成的。</p>
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:218]: 一并更新链接：

```suggestion
<p>ECMAScript 2015 引入了<a href="/zh-CN/docs/Web/JavaScript/Reference/Classes">class syntax</a> 到 JavaScript，作为使用更简单、更简洁的语法编写可重用类的一种方式，这更类似于 C++ 或 Java 中的类。在本节中，我们将个人和教师示例从原型继承转换为类，以向你展示它是如何完成的。</p>
```

File: files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html [hunk 6]
```diff
@@ -208,17 +208,17 @@ <h2 id="Object_member_summary">对象成员总结</h2>
  <li>那些定义在构造器函数中，用于给予对象实例的属性或方法。它们都很容易发现——在您自己的代码中，它们是构造函数中使用<code>this.x = x</code>类型的行；在浏览器内已经构建好的代码中，它们是可用于对象实例的成员（这些对象实例通常使用<code>new</code>关键字调用构造函数来创建，例如<code>var myInstance = new myConstructor()</code>）。</li>
  <li>那些直接在构造函数上定义，仅在构造函数上可用的属性或方法。它们通常仅在浏览器的内置对象中可用，并通过被直接链接到构造函数来识别，而不是实例。例如<code><a href="/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/keys">Object.keys()</a></code>。<strong>它们一般被称作静态属性或静态方法</strong>。</li>
  <li>那些在构造函数原型上定义，由所有实例和对象类继承的属性或方法。它们包括在构造函数的原型属性上定义的任何成员，如<code>myConstructor.prototype.x()</code>。</li>
- <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如<code>let teacher1 = new Teacher( 'Chris' );</code>，然后是 <code>teacher1.name</code>），也可以是一个对象文本（<code>let teacher1 = { name : 'Chris' }</code>，然后是<code>teacher1.name</code>）。</li>
+ <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如 <code>let teacher1 = new Teacher( 'Chris' );</code> 以及其属性 <code>teacher1.name</code>)，也可以是一个对象字面量（<code>let teacher1 = { name : 'Chris' }</code> 以及其属性 <code>teacher1.name</code>）。</li>
```
Comment (yin1999) [inline files/zh-cn/learn/javascript/objects/classes_in_javascript/index.html:211]:
```suggestion
 <li>那些在对象实例上可用的对象，它可以是像我们上面看到的构造函数实例化时创建的对象（例如 <code>let teacher1 = new Teacher( 'Chris' );</code> 以及其属性 <code>teacher1.name</code>），也可以是一个对象字面量（<code>let teacher1 = { name : 'Chris' }</code> 以及其属性 <code>teacher1.name</code>）。</li>
```

**Pull request merged**